packages/jsii-dotnet-runtime/src/Amazon.JSII.Runtime/jsii-runtime/jsii-runtime.js

```diff
@@ -0,0 +1,2 @@
O.Tar=R;const j=e=>e.isFile()?"File":e.isDirectory()?"Directory":e.isSymbolicLink()?"SymbolicLink":"Unsupported";e.exports=O},function(e,t,i){"use strict";const n=["|","<",">","?",":"],r=n.map(e=>String.fromCharCode(61440+e.charCodeAt(0))),s=new Map(n.map((e,t)=>[e,r[t]])),o=new Map(r.map((e,t)=>[e,n[t]]));e.exports={encode:e=>n.reduce((e,t)=>e.split(t).join(s.get(t)),e),decode:e=>r.reduce((e,t)=>e.split(t).join(o.get(t)),e)}},function(e,t,i){"use strict";const n=i(7),r=i(8),s=i(22),o=(i(18),i(2)),a=i(11),c=i(25),h=i(0),l=i(10),u=(e.exports=((e,t,i)=>{const n=r(e);if(!n.file)throw new TypeError("file is required");if(n.gzip)throw new TypeError("cannot append to compressed archives");if(!t||!Array.isArray(t)||!t.length)throw new TypeError("no files or directories specified");return t=Array.from(t),n.sync?u(n,t):d(n,t,i)}),(e,t)=>{const i=new s.Sync(e);let r,a,c=!0;try{try{r=o.openSync(e.file,"r+")}catch(t){if("ENOENT"!==t.code)throw t;r=o.openSync(e.file,"w+")}const s=o.fstatSync(r),h=n.alloc(512);e:for(a=0;a<s.size;a+=512){for(let e=0,t=0;e<512;e+=t){if(t=o.readSync(r,h,e,h.length-e,a+e),0===a&&31===h[0]&&139===h[1])throw new Error("cannot append to compressed archives");if(!t)break e}let t=new l(h);if(!t.cksumValid)break;let i=512*Math.ceil(t.size/512);if(a+i+512>s.size)break;a+=i,e.mtimeCache&&e.mtimeCache.set(t.path,t.mtime)}c=!1,f(e,i,a,r,t)}finally{if(c)try{o.closeSync(r)}catch(e){}}}),f=(e,t,i,n,r)=>{const s=new a.WriteStreamSync(e.file,{fd:n,start:i});t.pipe(s),p(t,r)},d=(e,t,i)=>{t=Array.from(t);const r=new s(e),c=new Promise((i,s)=>{r.on("error",s);let c="r+";const h=(u,f)=>u&&"ENOENT"===u.code&&"r+"===c?(c="w+",o.open(e.file,c,h)):u?s(u):void o.fstat(f,(c,h)=>{if(c)return s(c);((t,i,r)=>{const s=(e,i)=>{e?o.close(t,t=>r(e)):r(null,i)};let a=0;if(0===i)return s(null,0);let c=0;const h=n.alloc(512),u=(n,r)=>{if(n)return s(n);if((c+=r)<512&&r)return o.read(t,h,c,h.length-c,a+c,u);if(0===a&&31===h[0]&&139===h[1])return s(new Error("cannot append to compressed archives"));if(c<512)return s(null,a);const f=new l(h);if(!f.cksumValid)return s(null,a);const d=512*Math.ceil(f.size/512);return a+d+512>i?s(null,a):(a+=d+512)>=i?s(null,a):(e.mtimeCache&&e.mtimeCache.set(f.path,f.mtime),c=0,void o.read(t,h,0,512,a,u))};o.read(t,h,0,512,a,u)})(f,h.size,(n,o)=>{if(n)return s(n);const c=new a.WriteStream(e.file,{fd:f,start:o});r.pipe(c),c.on("error",s),c.on("close",i),m(r,t)})});o.open(e.file,c,h)});return i?c.then(i,i):c},p=(e,t)=>{t.forEach(t=>{"@"===t.charAt(0)?c({file:h.resolve(e.cwd,t.substr(1)),sync:!0,noResume:!0,onentry:t=>e.add(t)}):e.add(t)}),e.end()},m=(e,t)=>{for(;t.length;){const i=t.shift();if("@"===i.charAt(0))return c({file:h.resolve(e.cwd,i.substr(1)),noResume:!0,onentry:t=>e.add(t)}).then(i=>m(e,t));e.add(i)}e.end()}},function(e,t,i){"use strict";const n=i(12),r=(i(14).EventEmitter,i(18)),s=i(2),o=i(11),a=i(0),c=i(93),h=(c.sync,i(37)),l=Symbol("onEntry"),u=Symbol("checkFs"),f=Symbol("isReusable"),d=Symbol("makeFs"),p=Symbol("file"),m=S
```

Comment:
I don't think this file is supposed to be checked in

Reply:
No it is not. What the...